src/mon/MonCommands.h

```diff
@@ -155,6 +155,7 @@ COMMAND("pg ls-by-osd " \
 	"list pg on osd [osd]", "pg", "r", "cli,rest")
 COMMAND("pg ls " \
         "name=pool,type=CephInt,req=false " \
+        "name=osd,type=CephInt,req=false " \
```

Comment:
yeah, i know the trick :) thank you for your detailed explanation!
if `ceph` command accepts named arguments, maybe we could change it from positional argument to named argument. anyway, i will drop the second commit for now.